components/spinner/spinner.scss

```diff
@@ -110,3 +110,51 @@
     transform: rotate(360deg);
   }
 }
+
+.ui-spinner__loading-section {
+  align-items: center;
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  position: absolute;
+  width: 100%;
+  z-index: var(--tx-spinner-wrapper-z-index);
+}
+
+.ui-spinner__loading-text {
+  color: var(--tx-generic-color-primary);
+  font-family: var(--tx-generic-font-primary-font-family);
+  font-size: var(--tx-spinner-text-font-size);
+  font-weight: bold;
+  margin-left: 8px;
+}
+
+.ui-spinner__wrapper {
+  border-radius: var(--tx-spinner-wrapper-border-radius);
+  overflow: hidden;
+  position: relative;
+}
+
+.ui-spinner__content {
+  transition: all 0.2s ease-in-out;
+}
+
+.ui-spinner__wrapper_loading {
+  .ui-spinner__content:before {
+    background: var(--tx-generic-color-secondary-light);
+    bottom: 0;
+    content: "";
+    left: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+    transition: all 0.3s;
+    z-index: var(--tx-spinner-content-z-index);
```

Comment:
The good ol' logic of applying variables: is thing changing per brand? If not, keep on the component.

Reply:
Fixed :)

Reply:
@froskie The prop 'loading' needed when you use the component as a wrapper. It's 'loading' by default, but then when you finish loading your stuff inside the container, you can change it to hide spinner itself and show your content inside the wrapper. So that you don't need to implement it each time. When using component as a normal spinner without children, you won't need to change the prop.